farmhash.BUILD

```diff
@@ -15,7 +23,13 @@ genrule(
 cc_library(
     name = "farmhash",
     srcs = [prefix_dir + "/src/farmhash.cc"],
-    hdrs = [prefix_dir + "/src/farmhash.h"] + [":configure"],
+    # Don't run configure on Windows
+    hdrs = [prefix_dir + "/src/farmhash.h"] + select({
+        ":windows": [],
+        "//conditions:default": [":configure"],
+    }),
+    # Disable __builtin_expect support
+    copts = ["/DFARMHASH_OPTIONAL_BUILTIN_EXPECT"],
```

Comment:
Should this copt be Windows-only? (I'd have guessed that gcc would balk at a flag like this....)